examples/tests/valid/json.w

```diff
@@ -0,0 +1,24 @@
+let json_string  = Json "hello";
```

Comment:
As part of #1468 we should create an alias for this `json`, which will be beautiful to signal the fact that we think about it as a primitive type in our little cloud language.